Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/InvokeDelegateBuilder.cs

```diff
@@ -60,14 +61,19 @@ public InvokeDelegateBuilder(InternalLogger logger, HandlerInfo handler, MethodI
         /// <param name="customerObject">Wrapped customer object.</param>
         /// <param name="customerSerializerInstance">Instance of lambda input & output serializer.</param>
         /// <returns>Action delegate pointing to customer's handler.</returns>
-        public Action<Stream, ILambdaContext, Stream> ConstructInvokeDelegate(object customerObject, object customerSerializerInstance)
+        public Action<Stream, ILambdaContext, Stream> ConstructInvokeDelegate(object customerObject, object customerSerializerInstance, bool isPreJit)
         {
             var inStreamParameter = Expression.Parameter(Types.StreamType, "inStream");
             var outStreamParameter = Expression.Parameter(Types.StreamType, "outStream");
             var contextParameter = Expression.Parameter(typeof(ILambdaContext), "context");
 
             _logger.LogDebug($"UCL : Constructing input expression");
             var inputExpression = BuildInputExpressionOrNull(customerSerializerInstance, inStreamParameter, out var iLambdaContextType);
+            if (isPreJit)
+            {
+                _logger.LogInformation("PreJit: inputExpression");
```

Comment:
I think this should rather be logging at debug level to reduce chances of cluttering the function's CW logs for information it may not have asked for - ditto for other places in this PR. Do you think there's value for this as `info` over `debug`? Would this only end up logging if `LAMBDA_RUNTIMESUPPORT_DEBUG` is set?

Reply:
This logs only show up if `LAMBDA_RUNTIMESUPPORT_DEBUG` is set. They are just visible for developers that want to debug Amazon.Lambda.RuntimeSupport. They won't be visible to normal invokes unless the environment variable is set.